pkgs/tools/networking/dd-agent/datadog-agent.nix

```diff
@@ -35,7 +35,7 @@ let
     cmakeFlags = ["-DBUILD_DEMO=OFF" "-DDISABLE_PYTHON2=ON"];
   };
 
-in buildGoModule rec {
+in buildGo118Module rec {
```

Comment:
Please note that a previous PR that unpinned the version of go here broke the package due to a transitive dep.

Reply:
Can we use `buildGo119Module`? upstream seems to use go 1.19 https://github.com/DataDog/datadog-agent/blob/7.45.1/.go-version

Reply:
Unfortunately, we cannot. That results in the same issue with the transitive dep:

```
[flurie@a-m:~/code/github/flurie/nixpkgs]$ ./result/bin/agent 
panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.19 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.19 set. Notably, if go1.19 adds a moving garbage collector, this program is unsafe to use.
```